### PR TITLE
Radio buttons

### PR DESCRIPTION
**Pull request by soerenbf**
## Purpose

Updated the app to use radio buttons for option selection instead of button groups. The reason for this update is, to use best practice native form functionality to capture user input as much as possible.

## Changes

- Removed `ButtonGroup` component
- Added `Radios`
- Updated references to `ButtonGroup` to `Radios`
- Fixed a bug where it wasn't possible to go back to building release schedule from `SubmitTransaction`

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.

### CODE REVIEW

File: app/pages/multisig/AccountTransactions/UpdateBakerRestakeEarnings.tsx [hunk 1]
```diff
@@ -36,6 +35,7 @@ import { isMultiSig } from '~/utils/accountHelpers';
 import Label from '~/components/Label';
 
 import styles from './MultisignatureAccountTransactions.module.scss';
+import Radios from '~/components/Form/Radios';
```
Comment (shjortConcordium) [inline app/pages/multisig/AccountTransactions/UpdateBakerRestakeEarnings.tsx:38]: :angry: 
(probably doesn't matter though)

File: app/pages/Accounts/AccountDetailsPage/BuildSchedule/BuildSchedule.tsx [hunk 2]
```diff
@@ -18,11 +18,11 @@ import { BuildScheduleDefaults } from '~/components/BuildSchedule/util';
 import { scheduledTransferCost } from '~/utils/transactionCosts';
 import TransferView from '~/components/Transfers/TransferView';
 import DisplayEstimatedFee from '~/components/DisplayEstimatedFee';
-import ButtonGroup from '~/components/ButtonGroup';
 import { chosenAccountInfoSelector } from '~/features/AccountSlice';
 import ErrorMessage from '~/components/Form/ErrorMessage';
 
 import styles from './BuildSchedule.module.scss';
+import Radios from '~/components/Form/Radios';
```
Comment (shjortConcordium) [inline app/pages/Accounts/AccountDetailsPage/BuildSchedule/BuildSchedule.tsx:25]: angry
(probably doesn't matter though)

File: app/pages/Accounts/AccountDetailsPage/AccountDetailsPage.tsx [hunk 3]
```diff
@@ -53,7 +57,8 @@ export default withAccountSync(function DetailsPage() {
 
     const isBaker = Boolean(accountInfo?.accountBaker);
     const canTransfer = hasCredentials && Boolean(accountInfo);
-    const accountChanged = !useIsSubsequentRender();
+    const isFirstRender = !useIsSubsequentRender();
+    const accountHasChanged = isFirstRender && !hasLocationState;
```
Comment (shjortConcordium) [inline app/pages/Accounts/AccountDetailsPage/AccountDetailsPage.tsx:61]: This change does so we stay in the build schedule view, when changing account, which I thought we didn't want.

Comment (soerenbf) [reply app/pages/Accounts/AccountDetailsPage/AccountDetailsPage.tsx]: Right you are.. sorry! Fixed in newest commit

**Review by shjortConcordium — APPROVED** [review]

**Pull request merged**